subworkflows/local/pangenome_analysis.nf

```diff
@@ -0,0 +1,71 @@
+include { ROARY } from '../../modules/nf-core/roary/main'
+include { PIRATE } from '../../modules/nf-core/pirate/main'
+include { PANAROO_RUN } from '../../modules/nf-core/panaroo/run/main'
+include { PEPPAN } from '../../modules/local/PEPPAN/main'
+
+workflow PANGENOME_ANALYSIS {
+
+
+    take:
+    // should be ch_gff from prokka output
+    //ch_gff
+
+    main:
+
+    ch_versions = Channel.empty()
+
+    ch_gff = [
+        [ id:'pan_genome', single_end:false ],
+        [
+            file("/scratch/mzanwar/nf-seqqc/PROKKA_GCA_010673125.1/PROKKA_GCA_010673125.1.gff"),
+            file("/scratch/mzanwar/nf-seqqc/PROKKA_GCA_010798115.1/PROKKA_GCA_010798115.1.gff")
+
+        ]
+    ]
```

Comment:
Should remove hard coding of the input paths once the workflow input is defined

Reply:
updated with [d4add19](https://github.com/cidgoh/nf-seqqc/pull/58/commits/d4add191e949961678f503285af315f16f79bc29)